src/components/Board/Output/Output.container.js

```diff
@@ -179,6 +181,16 @@ export class OutputContainer extends Component {
     this.clearOutput();
   };
 
+  handleCopyClick = () => {
+    const { intl, showNotification } = this.props;
+    let labels = [];
+    this.props.output.map(symbol => {
```

Comment:
remove function curly brackets if map does contain a single line

Reply:
Got it, I think I fixed it with the most reent commit